### PR TITLE
Add statuses documentation

### PR DESCRIPTION
**Pull request by mattvagni**

### CODE REVIEW

File: statuses.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+When you have a support request that requires further investigation or work on your side you can set a thread to this status.
+
+This is particularly useful when you have support request that require longer technical investigation or research before getting back to the customer or marking it as done.
+
+## Close the Loop
+
+<Frame>![Todo / Close the Loop](/public/images/statuses/todo-close-the-loop.png)</Frame>
+
+If you use our [Linear Integration](/integrations/linear) then threads will move to this status when there was an update to a linked Linear issue.
```
Comment (jesushernandez) [inline statuses.mdx:31]: Let's also mention resolved thread discussions here.

File: statuses.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Waiting for Customer
+
+<Frame>
+  ![Snoozed / Waiting for Customer](/public/images/statuses/snoozed-waiting-for-customer.png)
+</Frame>
+
+Threads in this status are waiting for the customer to reply. By default, threads will be automatically moved to this status when you reply to them. You can change this setting in **Settings** → **Workflow**.
+
+When a customer replies, threads in this status will be moved to Needs Next Response.
```
Comment (jesushernandez) [inline statuses.mdx:41]:
```suggestion
When a customer replies, threads in this status will be moved to *Needs Next Response*.
```
Some emphasis, maybe?

File: statuses.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Paused for Later
+
+<Frame>![Snoozed / Paused for Later](/public/images/statuses/snoozed-paused-for-later.png)</Frame>
+
+When you need to pick up a thread later you can pause it for any amount of time. Once paused threads will have this status.
```
Comment (jesushernandez) [inline statuses.mdx:47]:
```suggestion
When you need to pick up a thread later, you can pause it for any amount of time. Once paused, threads will have this status.
```

File: statuses.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Once the pause duration you set runs out, or if the customer sends a new message, the thread will be moved back to it's previous status.
```
Comment (jesushernandez) [inline statuses.mdx:49]:
```suggestion
When the pause duration you set runs out, or if the customer sends a new message, the thread will be moved back to its previous status.
```
'Once' is already in the previous sentence.

File: statuses.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+## Done
+
+<Frame>![Done](/public/images/statuses/done.png)</Frame>
+
+When a customer issue is resolved, you can mark a thread as done. If a customer sends a new message the thread will move back to Needs Next Response.
```
Comment (jesushernandez) [inline statuses.mdx:55]:
```suggestion
When a customer issue is resolved, you can mark a thread as done. If a customer sends a new message, the thread will move back to *Needs Next Response*.
```

**Review by MitchellPetrie — COMMENTED** [review]
Small notes otherwise LGTM

File: statuses.mdx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+## Waiting for Customer
+
+<Frame>
+  ![Snoozed / Waiting for Customer](/public/images/statuses/snoozed-waiting-for-customer.png)
+</Frame>
+
+Threads in this status are waiting for the customer to reply. By default, threads will be automatically moved to this status when you reply to them. You can control this behaviour by going to **Settings** → **Workflow**
+
+When a customer replies, threads in this status will be moved to Needs Next Response.
```
[the remark this answers is not in the document]
Comment (mattvagni) [reply statuses.mdx:41]: I think with docs you have to assume people don't read it start to finish so this is helpful imo even if slightly repetitive.

[the remark this answers is not in the document]
Comment (MitchellPetrie) [reply statuses.mdx:41]: Fair 👍

**Review by andrewblaney — APPROVED** [review]

**Review by jesushernandez — APPROVED** [review]

**Pull request merged**